xinference/model/image/stable_diffusion/core.py

```diff
@@ -416,8 +416,12 @@ def _get_scheduler(model: Any, sampler_name: str):
     @contextlib.contextmanager
     def _reset_when_done(model: Any, sampler_name: str):
         assert model is not None
+        name_or_path: Optional[str] = getattr(model, "_name_or_path", None)
+        is_flux_sampler = "flux" in name_or_path.casefold() if name_or_path else False
```

Comment:
I think we'd better judge flux by `model_spec`,  so we may set this method to normal function instead of static method. Because we cannot ensure that the model path contains flux.